applications/CableNetApplication/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
```

Comment:
I updated the app-generator in #5385 
This changes this file a lot

Could you please use this branch to re-generate this file?

Sorry for the inconvenience, should not take much time now and will save you more work later

Reply:
how can I re-generate a single file with the app-generator?

Reply:
Just generate a new app and cooy-paste the file